Pages/Programming.md

```diff
@@ -34,6 +34,11 @@
 
 - [W3 HTML Introduction](https://www.w3schools.com/html/)
 
+### Python
+
+- [RealPython](https://realpython.com/)
```

Comment:
A description to `RealPython` could be added to give a glimpse what it is while also keeping consistent with the list.

Reply:
Done